subprojects/core/build.gradle.kts

```diff
@@ -186,14 +187,14 @@ dependencies {
 
     annotationProcessor(project(":internal-instrumentation-processor"))
     annotationProcessor(platform(project(":distributions-dependencies")))
+
+    testAnnotationProcessor(project(":internal-instrumentation-processor"))
+    testAnnotationProcessor(platform(project(":distributions-dependencies")))
 }
 
 strictCompile {
     ignoreRawTypes() // raw types used in public API
-}
-tasks.compileJava {
-    // Without this, javac will complain about unclaimed annotations
-    options.compilerArgs.add("-Xlint:-processing")
+    ignoreAnnotationProcessing() // Without this, javac will complain about unclaimed annotations
```

Comment:
❔ Should we mention that the `NonNullApi` annotation is problematic?

Reply:
I saw that javac shows a warning because of the `NonNullApi` annotation (maybe also for some other?). So I was wondering if we should add to a comment that javac complains that `NonNullApi` annotation is unclaimed. 

Although maybe there is some additional annotation that is unclaimed.